cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1 @@
+<!-- describe each component and how they work together -->
```

Comment:
It would be best to put at least 1 how-to on this page if it's going to be linked to from the left nav or from any other page in the docs (if not, then this is fine).

But if you do at at least one how-to, add something at the end like:

Additional How-to's will be added soon.

Reply:
Added some how-to's and message at the end